### PR TITLE
Make changes to code style

### PR DESCRIPTION
**Pull request by dymil**
I felt that this pull request ought to be separate from the one just affecting documentation.

There are some inconsistencies which I didn't address, like the placement of braces in function definitions.

The existing (non-Gurobi) unit tests pass, but I don't think those cover this, though most changes should be obviously correct, with the most notable exception being changing the types of `psrf_check` and `parallelism`.

### CODE REVIEW

File: dingo/volestipy.pyx [hunk 1]
```diff
@@ -18,7 +18,6 @@ import os
 import sys
 import numpy as np
 cimport numpy as np
-from libcpp cimport bool
```
Comment (vissarion) [inline dingo/volestipy.pyx:21]: are we sure this is not needed @hariszaf @TolisChal ?

**Review by vissarion — COMMENTED** [review]
Thanks @dymil for this PR! 

I am OK with most of the changes. Not sure about one though, AFAIR the choice of `int` instead of `bool` in the binding were because of an issue of Cython to not correctly handle bool types. 

@hariszaf @TolisChal can you confirm that?

**Comment by vissarion** [comment]
@dymil there are also some conflicts you have to resolve, thanks!

**Comment by dymil** [comment]
> AFAIR the choice of `int` instead of `bool` in the binding were because of an issue of Cython to not correctly handle bool types.

If I did it right, I preserved as much of the behavior as possible, even within each function. For example, [`bool(parallelism)`](https://docs.python.org/3/library/functions.html#bool) will be `True` or `False` as if it were the condition in the old if statements. It will be then be 1 or 0 accordingly [as a `bint`](https://cython.readthedocs.io/en/latest/src/userguide/pyrex_differences.html#boolean-int-type-e-g-it-acts-like-a-c-int-but-coerces-to-from-python-as-a-boolean), and that C int will then be implicitly converted to the C++ bool `parallelism` according to [the usual rules](https://en.cppreference.com/w/cpp/language/implicit_conversion#Boolean_conversions).

As an example which captures the above behaviour up until the call from Cython to C++:

```Cython
from cpython cimport bool
cdef bint a = bool(3)
print(a, <int>(a))
a = bool(None)
print(a, <int>(a))
```

produces

```
True 1
False 0
```

When I `print(check_psrf)` in `slow_mmcs()` in volestipy.pyx and when I `std::cout<<mmcs_set_of_parameters.psrf_check` in `mmcs_initialize()` in bindings.cpp, things seemed to behave as I expect, i.e., `None` becomes `False` becomes 0 when I print the bint and bool, respectively, and `[3]` becomes `True` becomes 1.

File: dingo/volestipy.pyx (hunk 1 above)
Comment (hariszaf) [reply dingo/volestipy.pyx:21]: i think it's not needed. we struggled a lot with the boolean vars and most probably it's the following line
```
from cpython cimport bool
```
that does the work.  

if the tests do not fail,  i guess it's fine. apologies for not having a clearer answer.

**Comment by vissarion** [comment]
Thanks @dymil for the explanations! Please merge with develop to see the tests running on this PR.

**Pull request merged**